wiki/Medals/Unlock_requirements/Dedication/id.md

```diff
@@ -0,0 +1,34 @@
+# Persyaratan untuk mendapatkan medali Dedication
+
+## osu!
+
+| Nama medali | Persyaratan | Deskripsi |
+| :-- | :-- | :-- |
+| 5,000 Plays | Raih jumlah main sebanyak 5.000 kali bermain pada mode permainan osu!. | There's a lot more where that came from. |
+| 15,000 Plays | Raih jumlah main sebanyak 15.000 kali bermain pada mode permainan osu!. | Must.. click.. circles.. |
+| 25,000 Plays | Raih jumlah main sebanyak 25.000 kali bermain pada mode permainan osu!. | There's no going back. |
+| 50,000 Plays | Raih jumlah main sebanyak 50.000 kali bermain pada mode permainan osu!. | You're here forever. |
+
+## osu!taiko
+
+| Nama medali | Persyaratan | Deskripsi |
+| :-- | :-- | :-- |
+| 30,000 Drum Hits | Raih jumlah akumulatif pukulan drum sebanyak 30.000 pukulan pada mode permainan osu!taiko. | Did that drum have a face? |
```

Comment:
`akumulatif` and `total` is standing to one, suggest to just delete `akumulatif`

Reply:
I was really confused how to tell the drum hit, note key, and caught fruit count achievement doesn't have to be achieved in single play.